fake-ipa/fake_ipa/heartbeater.py

```diff
@@ -91,13 +91,18 @@ def _heartbeat_expected(self, agent, previous_heartbeat):
 
     def do_heartbeat(self, system, agent):
         """Send a heartbeat to Ironic."""
+
+        # if tls enabled with fakeIPA use HTTPS else HTTP
+        adv_protocol="http"
+        if  self._config.get("FAKE_IPA_CERTFILE") and self._config.get("FAKE_IPA_KEYFILE"):
```

Comment:
Should we also verify that these files exist? Also, pep8 dictates that we should check `if foo is not None:` instead of `if foo:`, as there are many kinds of falsy values in python.

Reply:
done thanks